xwiki-platform-core/xwiki-platform-notifications/xwiki-platform-notifications-api/src/main/java/org/xwiki/notifications/internal/CachedModelBridge.java

```diff
@@ -95,6 +96,13 @@ public void setStartDateForUser(DocumentReference userReference, Date startDate)
     }
 
     @Override
+    public PageNotificationEventDescriptor getPageNotificationEventDescriptor(DocumentReference documentReference)
+            throws NotificationException
+    {
+        return modelBridge.getPageNotificationEventDescriptor(documentReference);
```

Comment:
No, it’s only called when adding / editing / deleting a `PageNotificationEventDescriptorClass` XObject, so I don’t think that it’s relevant enough.